### ISSUE TITLE
Improve `mix test` experience: speed up and silence monitor logs

### ISSUE THREAD

**Issue by pdobacz** [omisego/elixir-omg#634]
TL;DR from slack:

Some `mix test` tests are slowish/loudish:

```
OMG.EthereumClientMonitorTest
2019-04-19 10:35:00.097 [warn] module=OMG.EthereumClientMonitor function=handle_event/2 ⋅Health check raised :ethereum_client_connection alarm.⋅
2019-04-19 10:35:00.097 [warn] module=OMG.EthereumClientMonitor function=handle_event/2 ⋅Health check established connection to the client. :ethereum_client_connection alarm clearead.⋅
  * test that alarm gets raised if there's no ethereum client running and cleared when it's running2019-04-19 10:35:00.597 [warn] module=OMG.EthereumClientMonitor function=handle_event/2 ⋅Health check raised :ethereum_client_connection alarm.⋅
2019-04-19 10:35:01.099 [warn] module=OMG.EthereumClientMonitor function=handle_event/2 ⋅Health check established connection to the client. :ethereum_client_connection alarm clearead.⋅
  * test that alarm gets raised if there's no ethereum client running and cleared when it's running (1009.6ms)
  * test that we don't overflow the message queue with timers when Eth client needs time to respond (4993.8ms)
  * test that alarm gets raised if there's no ethereum client running2019-04-19 10:35:06.601 [warn] module=OMG.EthereumClientMonitor function=handle_event/2 ⋅Health check raised :ethereum_client_connection alarm.⋅
```

1. let's speed up the 3 slowest monitor tests (5s, 1s, 500ms), proposition: tweak the waiting/interval times by configuring to lower values in `:test` env's config
    - those aren't `:integration` tests (`geth` is mocked out) so should be fast and don't belong with such tag
    - it's good to keep them in the rapid `mix test` run if possible
2. let's **selectively** silence the `warn` logs coming out of `EthereumClientMontiror`, in `:test` env only, so that the output is clean on passing tests.
    - this improves the experience and makes us focus on anything that gets outputed during tests
    - we don't want to elevate these logs to `info`, as they're signalling something's off but recoverable, so `warn` is justified

**Comment by pdobacz** [comment]
UPDATE:

for `mix test --only integration` I'm also getting:

```
2019-04-19 14:01:50.105 [warn] module=OMG.RPC.Plugs.Health function=handle_event/2 ⋅Alarm boot_in_progress was raised.⋅
```

I'd say we either handle the same way as the `EthereumClientMonitor` warns. But what if these logs to silence become abundant?

**Comment by pdobacz** [comment]
warning logs and `mix test` duration is still problematic in 38f47e67c24f9808ffe9962d130756b85730d6b6, adding to our todo list in `Cinnabon` in case anyone has capacity.
